modules/secure-cloud-run-net/README.md

```diff
@@ -0,0 +1,79 @@
+# Secure Cloud Run Network
+
+This module handles the basic deployment network configurations for Cloud Run usage.
+
+The resources/services/activations/deletions that this module will create/trigger are:
+
+* Creates Firewall rules on your **VPC Project**.
```

Comment:
It seems like alot of these resources are gated by `connector_on_host_project`. We should add some docs around it (or point to the upstream doc) and how to make that decision.